### ISSUE TITLE
Major: Inventory item-> Incorrect/Random placed block, items changed/destroyed in players inventories when the latest version is loaded, and sounds are mixed up after upgrading worlds from 19w09a to 19w11b. Fix available.

### ISSUE THREAD

**Issue by Justsnoopy30** [FabricMC/fabric#122]
https://www.dropbox.com/s/eff3h2g4g57hzd7/Minecraft%2019w11b%202019-03-14%2018-47-28.mp4?dl=1

**Comment by Justsnoopy30** [comment]
![C__Users_EV Local_Desktop_Minecraft Server_Fabric_mods 3_14_2019 7_09_43 PM](https://user-images.githubusercontent.com/33374736/54399524-dc7c2500-468c-11e9-956d-304747ac575b.png)

**Comment by modmuss50** [comment]
After a lot of trail and error I have been able to reproduce this.

Fabric server with fabric api.
Vanilla client, does not happen with fabric client.

Most blocks work, it can be reproduced with the following data file:

https://ss.modmuss50.me/fabricRegistry.dat

Deleting that file fixes it, im not sure what caused the ids to break, possibly updating the world across versions?

**Comment by asiekierka** [comment]
```Deleting that file fixes it, im not sure what caused the ids to break, possibly updating the world across versions?
--```

Or adding and then removing a mod, *then* updating the world.

**Comment by modmuss50** [comment]
Should now be finally fixed in 1.16-pre1
